src/components/NewsFeed/MakePost.jsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import "./MakePost.css";
+import PropTypes from "prop-types";
+import firebaseWrapper from "../../firebaseWrapper";
+
+class MakePost extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      text: "",
+      photo: null,
+      likes: 0
```

Comment:
doesn't look like `likes` ever changes. You can probably remove this if that is true.